README.md

```diff
@@ -1 +1,129 @@
-# School-library-Ruby
\ No newline at end of file
+# School Library Project in Ruby<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 morse_code <a name="about-project"></a>
```

Comment:
Great job @yosaddis 🤝

- [ ] Kindly ensure your readme file is properly customized and is referencing the exact project you are working on. I believe you might need to change this project title from morse_code to School Library Project.